src/applications/simple-forms/21-0845/utils.js

```diff
@@ -55,3 +55,19 @@ export const getEnumsFromConstants = (constants = {}) => {
 
   return enums;
 };
+
+export const objHasEmptyValues = obj => {
```

Comment:
could you add unit test(s) for this?

Reply:
Ticket [#738](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/738) opened for unit-tests.  Turns out the whole utils.js is missing unit-tests, so I thought I'd handle this separately.